### ISSUE TITLE
Label the edges with weight values

### ISSUE THREAD

**Issue by bikashg** [alexlenail/NN-SVG#33]
Wondering if there is a way to label the edges connecting the nodes with the weight values.

**Comment by alexlenail** [comment]
Although I agree that  would be nice, it seems quite difficult to me to keep the numerals non-overlapping. What do you think?

**Comment by bikashg** [comment]
True. Fixing the labels without overlapping would be a significant challenge. Someone with appropriate skills could submit a pull request in future, hopefully. I keep it open for that sake but would also agree if you chose to close it.

**Comment by alexlenail** [comment]
I think it's a good feature to add, so I'll leave it open. Sadly, I don't have bandwidth right now to add it.
